web/public/icons/favicon.svg

```diff
@@ -1,24 +1,20 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="1024" height="1024" viewBox="0 0 1024 1024" fill="none">
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
   <style>
-    path,
-    circle {
+    path {
       fill: black;
     }
     @media (prefers-color-scheme: dark) {
-      path,
-      circle {
+      path {
         fill: white;
       }
```

Comment:
Don't this need one for the rect to fill as black when dark mode is preferred?

Reply:
I thought so too, but as it's just a clipping path it doesn't matter.